packages/react/src/modules/in-app-message/components/Banner/styles.css

```diff
@@ -1,57 +1,53 @@
-/* TODO: Set up css variables to match designs */
-/* TODO: Consider shared theme for all ootb components */
-/* TODO: maybe brink in telegraph tokens */
 .iam-banner {
   display: flex;
   align-items: center;
   justify-content: space-between;
-  background: #ffffff;
-  padding: 16px 24px;
-  border-radius: 8px;
-  border: 0.5px solid #e0e1e6;
-  box-shadow: 0px 12px 20px rgba(28, 32, 36, 0.1);
-  gap: 16px;
+  background: var(--iam-surface);
+  padding: var(--knock-spacing-4) var(--knock-spacing-6);
+  border-radius: var(--knock-rounded-4);
+  border: 0.5px solid var(--iam-border);
+  box-shadow: var(--knock-shadow-2);
+  gap: var(--knock-spacing-4);
```

Comment:
**Why knock and iam**
This partially stems from the structure of the designs. I think there are two levels of CSS variables:
1. Design tokens like our spacing and color scales (e.g. blue 1-12). These are the `knock` CSS variables which are based on Telegraph and could be used in anywhere in the React components. 
2. Semantic tokens like accent color which are used in the out of the box components.  The initial values come from the Knock variables (like we default the colors to orange which is our brand color, but could be swapped out for `--knock-green-*` for example). 

The semantic tokens don't cover all of the component styles (like spacing) which is why I'm using a mix of both. I see the semantic tokens as being there to make it easy to quickly re-theme the components like by changing colors. I think it'd be overly complex to capture all of the component styles in the semantic tokens. For example, I don't think there's a good way to capture really specific things like the banner flex gap in a CSS variable without introducing a ton of variables with hyper specific names. 

Right now, the semantic tokens (e.g. `--iam`) match 1:1 the variables in the public Figma designs. We could definitely adjust those if there are CSS properties we want to expose in that namespace.

**iam naming**
Yeah, I see how the name can be a bit confusing. Was mostly following what we do for the feed with `rnf`. I'll leave a note to discuss as a team when Chris is back just to land on something we all like. I think there's merit in adding `knock` to the beginning (`--knock-iam-accent`) or just using `knock` like (`--knock-accent`) depending if we want to scope the 'semantic' styles to just in app messaging (although realistically, when are we going to ship another class of components that would overlap with those styles?).

Reply:
> I see the semantic tokens as being there to make it easy to quickly re-theme the components like by changing colors.

Ahh right. So for example, when toggling between light mode vs dark mode dynamically, design token vars remain the same, but semantic token vars change? Or mapping their own design system tokens to semantic tokens. Thanks for explaining!

> I think there's merit in adding knock to the beginning (--knock-iam-accent) or just using knock like (--knock-accent) depending if we want to scope the 'semantic' styles

Yeah my rough thinking:
1) Any vars we are bringing in probably should have a consistent prefix clearly identifying or associating to "knock". Something like `knock-`. 
2) For design token I think we could potentially just use the secondary prefix/namespace of `tgph`? They are affectively the same telegraph vars with a `knock-` prefix like `--knock-tgph-text-4`. Since they aren't meant to be overwritten or used directly, it matters less what "tgph" means. 
3) For semantic tokens, we could get rid of the `iam` prefix and make it more readable. So `--knock-accent` or `--knock-color-accent`. 

Food for thoughts! And we can discuss and make a decision when Chris is back.

Reply:
1. For light/dark mode toggling, the Knock/telegraph tokens actually do change value, so for example `--knock-gray-1` goes from a light gray in light mode to a dark gray in dark mode. That way when we set something like `--iam-text-light: var(--knock-gray-1` (not an actual definition used atm, just an example), when you toggle light/dark mode the underlying values changes. 

2. I think that's a solid alternative as well!